quickjs.c

```diff
@@ -50512,7 +50513,8 @@ static JSValue js_dataview_setValue(JSContext *ctx,
 {
     JSTypedArray *ta;
     JSArrayBuffer *abuf;
-    int is_swap, size;
+    BOOL littleEndian, is_swap;
```

Comment:
I know it's nitpicking but for consistency I would suggest calling it `little_endian`.

Reply:
I used `littleEndian` for consistency with the ECMA document, otherwise I would definitely use `little_endian`.